site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+  </tr>
+  <tr>
+     <td><code>client_metadata_endpoint</code> (optional)</td>
+     <td>URL for the <a href="#client-metadata-endpoint">client metadata endpoint</a>.</td>
+  </tr>
+  <tr>
+     <td><code>id_assertion_endpoint</code> (required)</td>
+     <td>URL for the <a href="#id-assertion-endpoint">ID assertion endpoint</a>.</td>
+  </tr>
+  <tr>
+     <td><code>branding</code> (optional)</td>
+     <td>Object which contains various branding options.</td>
+  </tr>
+  <tr>
+     <td><code>branding.background_color</code> (optional)</td>
+     <td>Background color of the "Continue as..." button. Use the relevant CSS syntax, namely
+<a href="https://drafts.csswg.org/css-color-4/#typedef-hex-color"><code>hex-color</code></a>,
+<a href="https://drafts.csswg.org/css-color-5/#funcdef-hsl"><code>hsl()</code></a>,
+<a href="https://drafts.csswg.org/css-color-5/#funcdef-rgb"><code>rgb()</code></a>, or
+<a href="https://drafts.csswg.org/css-color-4/#typedef-named-color"><code>named-color</code></a>.</td>
+  </tr>
+  <tr>
+     <td><code>branding.color</code> (optional)</td>
+     <td>Branding option which sets the background color of the "Continue as..." button. Use the relevant CSS syntax, namely
+<a href="https://drafts.csswg.org/css-color-4/#typedef-hex-color"><code>hex-color</code></a>,
+<a href="https://drafts.csswg.org/css-color-5/#funcdef-hsl"><code>hsl()</code></a>,
+<a href="https://drafts.csswg.org/css-color-5/#funcdef-rgb"><code>rgb()</code></a>, or
+<a href="https://drafts.csswg.org/css-color-4/#typedef-named-color"><code>named-color</code></a>.</td>
+  </tr>
+  <tr>
+     <td><code>branding.icons</code> (optional)</td>
+     <td>Branding option which sets the icon object, displayed in the sign-in dialog. The icon object is an array with two parameters:
+        <ul>
+           <li><code>url</code> (required): URL of the icon image. This does not support SVG images.<li>
+           <li><code>size</code> (optional): icon dimensions, assumed by the application to be square and single resolution. This number must be greater than or equal to 25.</li>
+         </ul>
+      </td>
+  </tr>
+</table>
+
+{% Img
+   src="image/VbsHyyQopiec0718rMq2kTE1hke2/rFrfrCL0awt5zmyqvaM9.jpg", alt="How branding is applied to the FedCM dialog", width="600", height="332", class="screenshot"
+%}
+
+Here's an example response body from the IdP:
+
+```json
+{
+  "accounts_endpoint": "/accounts.php",
+  "client_metadata_endpoint": "/metadata.php",
+  "id_assertion_endpoint": "/id_assertions.php",
+  "branding": {
+    "background_color": "green",
+    "color": "0xFFEEAA",
+    "icons": [{
+      "url": "https://idp.example/icon.ico",
+      "size": 25
+    }]
+  }
+}
+```
+
+Once the browser fetches the config file, it sends subsequent requests to the
+IdP endpoints:
+
+{%
+  Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/8EzrPvIwOsuuzdKJUJ6E.png", alt="IdP endpoints", width="800", height="419", class="type--full-bleed"
```

Comment:
This image does not include the well-known file. Here's the most up to date flowchart: https://fedidcg.github.io/FedCM/#high-level-design

Reply:
Actually, this image is really handy and solves the text size issue as well. I'll borrow this one :D